packages/v4/src/content/design-guidelines/components/notification-badge/notification-badge.md

```diff
@@ -1,5 +1,29 @@
 ---
 id: Notification badge
 section: components
+related: ['Notification drawer', 'Page']
 ---
-Use a **Notification badge** when you want to apply a small badge to an icon. This is primarily used to indicate incoming notifications in the page header, but it can be useful for any use case where adding a badge to an icon is needed.
+The **Notification badge** is intended to be used with a [Notification drawer](/components/notification-drawer) as a visible indicator to alert the user about incoming notifications.
+
+## Notification badge elements
+
+<img src="./img/notification-badge.png" alt="basic notification drawer" width="598"/>
+
+**1. Icon:** the notification badge will be located in the masthead and display the 'fa-bell' icon in it's default state. Clicking the icon (or anywhere within the badge) will toggle the notification drawer open and closed.
```

Comment:
@mmenestr The problem was that the "**" to make the text bold was placed before the number rather than after. For some reason this changes the way the items indent. I changed this in both files. I think that we've been inconsistent about this in the past. I've probably done it both ways myself but happy to adopt this standard.